wpeverest/user-registration
# Tweak - Redirect url as slug in login and my account shortcode

### All Submissions:



### Changes proposed in this Pull Request:

 This PR gives a feature where the user can put a page slug in the redirect URL of user_registration_login and user_registration_my_account shortcode.
 Example :- [user_registration_my_account redirect_url = sample-page] or [user_registration_login = sample-page]

### How to test the changes in this Pull Request:

1. Create the two pages one using the login short code and the other using the my account shortcode with redirect url as page slug.
2. verify after login from both pages it is redirecting to that page or not.


### Types of changes:


### Other information:


### Changelog entry

> Tweak - Redirect url as slug in login and my account shortcode
